src/Containers.jsx

```diff
@@ -636,9 +640,9 @@ class Containers extends React.Component {
 
             filtered.sort((a, b) => {
                 // Show unhealthy containers first
-                if (this.props.containersDetails[a] && this.props.containersDetails[b]) {
-                    const a_health = this.props.containersDetails[a].State.Health || this.props.containersDetails[a].State.Healthcheck;
-                    const b_health = this.props.containersDetails[b].State.Health || this.props.containersDetails[b].State.Healthcheck;
+                if (this.props.containers[a] && this.props.containers[b]) {
+                    const a_health = this.props.containers[a].State.Health || this.props.containers[a].State.Healthcheck;
+                    const b_health = this.props.containers[b].State.Health || this.props.containers[b].State.Healthcheck;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230712-150126-d59c3ba5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)